### PR TITLE
HOTT-1304 News page

### PR DESCRIPTION
**Pull request by jebw**
### Jira link

[HOTT-1304](https://transformuk.atlassian.net/browse/HOTT-1304)

### What?

I have added/removed/altered:

- [x] Added a Latest News page
- [x] Added a News RSS feed
- [x] Linked the Latest News from the homepage
- [x] Updated the README to recommend updating the NEWS.yml file
- [x] Updated the README with missing deployment information

### Why?

I am doing this because:

- We want to be able to update our users with information about changes to the API

### CODE REVIEW

File: source/news.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  <link href="/feed.xml" type="application/atom+xml" rel="alternate" title="Latest News Atom feed" />
+<% end %>
+
+# Latest News
+
+Changes to the Trade Tariff API are listed below.
+
+These changes can be subscribed to at the [Atom (RSS) feed](/feed.xml)
+
+_All changes apply to the v2 APIs. The v1 API will not be updated from this point onwards._
```
Comment (willfish) [inline source/news.html.md.erb:14]: :clap:

File: NEWS.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+- date: 2021-12-20
+  title: Addition of "Rules of Origin" API
+  content: |
+    Addition of "Rules of Origin" API to illustrate the rules of origin
+    applicable to the import of goods under preferential tariff regimes.
+    This covers:
+
+    * product-specific rules applicable to each rules of origin scheme
+    * proofs of origin
+    * any relevant links and background information on the trading relationship
```
Comment (willfish) [inline NEWS.yml:10]: I assume this content is okay :+1:

**Review by willfish — APPROVED** [review]

**Review by alexdesi — APPROVED** [review]
🕺

File: NEWS.yml (hunk 2 above)
Comment (jebw) [reply NEWS.yml:10]: Yeh, came from @mattlavis-transform

**Pull request merged**